vector_getters.go

```diff
@@ -201,3 +201,49 @@ func (vec *vector) getSliceChild(offset uint64, length uint64) []any {
 	}
 	return slice
 }
+
+func (vec *vector) getUnion(rowIdx mapping.IdxT) any {
+	if vec.getNull(rowIdx) {
+		return Union{Tag: "", Value: nil}
+	}
+
+	// Get the tag vector (stored at index 0)
+	tagVector := mapping.StructVectorGetChild(vec.vec, 0)
+	if tagVector.Ptr == nil {
+		return Union{Tag: "", Value: nil}
+	}
```

Comment:
We should remove all `.Ptr == nil` checks from this function - these cases should never happen, and if they do, they are a bug.

Reply:
Thanks' for seeing that.